### PR TITLE
Adding the getPagesTranscludedBy() function

### PR DESCRIPTION
**Pull request by ValentinBrclz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,10 @@ Gets the list of pages in a given namespace - [read more](http://www.mediawiki.o
 
 Gets the list of pages by a given prefix - [read more](https://www.mediawiki.org/wiki/API:Allpages)
 
+### bot.getPagesTranscludedBy(page, callback)
+
+Gets the list of pages that are transcluded by the given page - [read more](https://www.mediawiki.org/wiki/API:Transcludedin)
```
Comment (macbre) [inline README.md]: Isn't the description a bit misleading here? The documentation says:

> Find all pages that transclude the given pages

Comment (ValentinBrclz) [inline README.md]: That's right, thank you for checking that and making it clearer. I just corrected it. :smile:

File: lib/bot.js [hunk 2]
```diff
@@ -300,6 +300,22 @@ module.exports = (function() {
 			});
 		},
 
+		getPagesTranscludedBy: function (template, callback) {
```
Comment (macbre) [inline lib/bot.js]: So shouldn't this function be called a bit different, let's say `getPagesWithTemplate`?

Comment (ValentinBrclz) [inline lib/bot.js]: What about ```getPagesTranscluding()```? The thing is that it is not necessary a template that is transcluded...

Comment (macbre) [inline lib/bot.js]: Sounds even better ;) :+1:

**Comment by ValentinBrclz** [comment]
Done! :smile:

**Comment by macbre** [comment]
Thanks and sorry for the delayed feedback

**Comment by ValentinBrclz** [comment]
:+1: No worries, we all have a life outside of github.

**Pull request merged**